src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalController.java

```diff
@@ -270,4 +272,60 @@ public ResponseEntity addUserToOrganisation(
 
         return inviteUserToOrganisation(newUserCreationRequest, organisationIdentifier, userId);
     }
+
+    @ApiOperation(
+            value = "Delete an Organisation",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            }
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    code = 204,
+                    message = "A representation of the Deleted organisation",
+                    response = DeleteOrganisationResponse.class
+            ),
+            @ApiResponse(
+                    code = 401,
+                    message = "Unauthorized Error : The requested resource is restricted and requires authentication"
+            ),
+            @ApiResponse(
+                    code = 403,
+                    message = "Forbidden Error: Access denied"
+            ),
+            @ApiResponse(
+                    code = 404,
+                    message = "Not Found"
+            ),
+            @ApiResponse(
+                    code = 500,
+                    message = "Internal Server Error"
+            )
+
+    })
+    @DeleteMapping(

```

Comment:
For this endpoint you may also need to include `@ResponseStatus(value = HttpStatus.NO_CONTENT)` otherwise Swagger will include an empty, default 200 response as the successful response when we do not have one.

Reply:
yes . i have updated